Tools/TestWebKitAPI/Tests/mac/BackgroundColor.mm

```diff
@@ -25,10 +25,13 @@
 
 #import "config.h"
 
+#import "CGImagePixelReader.h"
 #import "PlatformUtilities.h"
 #import "TestWKWebView.h"
+#import <WebCore/Color.h>
 #import <WebKit/WKWebViewPrivate.h>
 #import <wtf/RetainPtr.h>
+#import <wtf/text/TextStream.h>
```

Comment:
I think you originally added this header for debugging, but it seems like it can be removed now?